### PR TITLE
Add IE11 compatibility meta tag

### PR DESCRIPTION
**Pull request by mynnx**
Issue:

## What I did
I could get IE11 to render the Storybook, but it would not render the iframe view.  It kept defaulting to older document modes which were too outdated to render our JS.

The only way I could reliably get IE11 to render the iframe view was to add the following meta tag, which is present in the Storybook template:

```<meta content="IE=edge" http-equiv="X-UA-Compatible" />```

I'm not well-versed in how this works; my reading suggests that an HTTP header is usually preferred but we're working with `webpack-dev-server` and don't have a ton of control over that.

If anyone has suggestions about how this should work, I'm open to hearing them.

## How to test
Open the iframe view in IE11 with and without the meta tag; observe that it does not render without.

Is this testable with jest or storyshots?
Not unless we use Browserstack or Saucelabs

## Does this need a new example in the kitchen sink apps?
No

## Does this need an update to the documentation?
No

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
Adding the header on static builds will be super annoying too, so this is perfect actually!

Thank you for caring @mynnx !

**Comment by ndelangen** [comment]
Could I request you also perform this change for the Vue and Angular apps?

**Comment by mynnx** [comment]
@ndelangen Thanks for the quick response!  I tried following the instructions at https://github.com/storybooks/storybook/blob/master/docs/src/pages/configurations/add-custom-head-tags/index.md to add this tag, but found that it was not injected.  Can you verify that this doc page is still accurate and that the functionality still works?

**Comment by ndelangen** [comment]
Epic! Merge when ready

**Comment by mynnx** [comment]
@ndelangen I can't merge without write access and it appears that CI failed.  Anything you'd like me to do?

**Comment by ndelangen** [comment]
I'm looking into it, looks like an outage at circleCI.

I leave the "Merge when ready" remark, because I usually don't wait around for the CI to approve, and browse elsewhere. With that comment other maintainers know it was my intention to merge.

**Pull request merged**
